source/core/src/main/com/csse3200/game/components/screendisplay/LoseScreenDisplay.java

```diff
@@ -114,6 +131,11 @@ private void addActors() {
         Image playerDead;
 
         if(animals.contains(LastAttackAnimal)){
+            Image background = new Image(new Texture(Gdx.files.internal(backgroundPaths[animals.indexOf(LastAttackAnimal)])));
```

Comment:
not a big deal but this texture never gets disposed of, could impact performance if you die repeatedly. You could use the resource service to manage textures or just add a call to dispose the texture somewhere

Reply:
Resolved. I saved the texture into a variable and disposed it in the dispose() function.